src/translations/data.json

```diff
@@ -3153,7 +3153,7 @@
   },
   "wordErrorInfo": {
     "en": "Select the type of mistake and write what you propose to correct.",
-    "isv": "Izberite tip pogrěšky i napišite, čto prědlagajete popraviti.",
+    "isv": "Izberi tip pogrěšky i napiši, čto prědlagaješ popraviti.",
```

Comment:
Začto? Forma na vy jestvuje bez malogo vo vsih jezykah, ne?

Reply:
Radi jednakovosti. Ok, sdělaju obratno.